commands/proposal_submission.go

```diff
@@ -536,14 +536,14 @@ func checkUpdateFuture(future *types.UpdateFutureProduct) Errors {
 		errs.AddForProperty("proposal_submission.terms.change.update_market.changes.instrument.product.future.quote_name", ErrIsRequired)
 	}
 
-	errs.Merge(checkOracleSpec(future.OracleSpecForSettlementPrice, "oracle_spec_for_settlement_price", "proposal_submission.terms.change.update_market.changes.instrument.product.future"))
-	errs.Merge(checkOracleSpec(future.OracleSpecForTradingTermination, "oracle_spec_for_trading_termination", "proposal_submission.terms.change.update_market.changes.instrument.product.future"))
+	errs.Merge(checkOracleSpec(future.DataSpecForSettlementPrice, "oracle_spec_for_settlement_price", "proposal_submission.terms.change.update_market.changes.instrument.product.future"))
```

Comment:
this might introduce breaking changes on front end and system-tests, but I think we should update the name of the property in the error messages as follow:

```suggestion
	errs.Merge(checkOracleSpec(future.DataSpecForSettlementPrice, "data_spec_for_settlement_price", "proposal_submission.terms.change.update_market.changes.instrument.product.future"))
```

Reply:
Yes, I added this in my latest changes https://github.com/vegaprotocol/vega/pull/5980/commits/49a728b85fc2470646ecc9d89c411d759db9b56f#diff-872de7073d1b2054247d4d9bfcdc63c49d530f5ee501083b4fc7ec970ce13051L519-R537
Thanks!